notify/email/email.go

```diff
@@ -123,11 +131,21 @@ func (n *Email) auth(mechs string) (smtp.Auth, error) {
 }
 
 // Notify implements the Notifier interface.
-func (n *Email) Notify(ctx context.Context, as ...*types.Alert) (bool, error) {
+func (n *Email) Notify(ctx context.Context, as ...*types.Alert) (ok bool, err error) {
+	ctx, span := tracer.Start(ctx, "email.Email.Notify", trace.WithAttributes(
+		attribute.Int("alerts", len(as)),
```

Comment:
Let's move this up attribute up into `dispatch.go` instead?

Reply:
it's already here actually: https://github.com/prometheus/alertmanager/pull/3673/files#diff-71b3813922f4c2abc260ff5be5e31cf64eeb31eb3d7272f19f4fa889fc3b27d5R93 so it could just be dropped...